QuickFixers/Views/Shared/_SPLayout.cshtml

```diff
@@ -0,0 +1,49 @@
+﻿<!DOCTYPE html>
+<html>
```

Comment:
Work with @BSaenz12 to resolve nav bar logic and conflicts.

Reply:
In-Progress...